src/wms/Database.ts

```diff
@@ -49,6 +51,9 @@ export class Database {
       .where('event_date', '=', dateformat(exportDate, 'yyyy-mm-dd'));
       // TODO: add filtrering for 5 vtfs
       // .havingIn('ngt_site.site_id', [])
+
+    console.info('getstaffSchedules ending');
```

Comment:
maybe say 'getstaffSchedules successful'? is there a case where this method can fail?

Reply:
You could get some events that have failed. I'm happy with 'ending' which doesn't indicate any success or failure. The purpose of the log entry is to indicate that the method has run and is about to exit. Handy to determine where any timeouts occur.

Reply:
Is it worth having the method in a try catch so you can handle the failure scenario?

Reply:
Failure to read the database is terminal and is captured in the handler try/catch. It does raise the question about whether we want a retry policy. I'll add a ticket for that.